src/Validators/BaseValidator.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace OpenActive\Validators;
+
+class BaseValidator implements ValidatorInterface
+{
+    /**
+     * Coerce given value to the type the validator is validating against.
+     *
+     * @param mixed $value The value to coerce.
+     * @return mixed The same value.
+     */
+    public function coerce($value)
+    {
+        // For all the classes that inherit BaseValidator
+        // that don't need to coerce,
+        // This is a no-op
+        return $value;
+    }
+
+    /**
+     * Run validation on the given value.
+     *
+     * @param mixed $value The value to validate.
+     * @return bool Whether validation passes or not.
+     */
+    public function run($value)
+    {
+        return false;
+    }
+
+    /**
+     * Returns a validator instance from a given type name.
+     *
+     * @param string The type name
+     * @return \OpenActive\Validators\ValidatorInterface
+     */
+    public static function getValidator($type)
+    {
+        $isTypeArray = false;
```

Comment:
Could do `$isTypeArray = substr($type, -2) === "[]"`?

Reply:
Resolved in https://github.com/openactive/models-php/pull/11/commits/745c1084d640774050acf4f51073f12b97f6f811